client/firewall/uspfilter/uspfilter.go

```diff
@@ -224,33 +224,33 @@ func (m *Manager) dropFilter(packetData []byte, rules []Rule, isIncomingPacket b
 		log.Errorf("unknown layer: %v", d.decoded[0])
 		return true
 	}
+
+	var ip net.IP
+	switch ipLayer {
+	case layers.LayerTypeIPv4:
+		if isIncomingPacket {
+			ip = d.ip4.SrcIP
+		} else {
+			ip = d.ip4.DstIP
+		}
+	case layers.LayerTypeIPv6:
+		if isIncomingPacket {
+			ip = d.ip6.SrcIP
+		} else {
+			ip = d.ip6.DstIP
+		}
+	}
+
+	if len(rules["0.0.0.0"]) > 0 || len(rules["::"]) > 0 {
+		return false
```

Comment:
I'm not sure that it is correct. Because we decide to not drop packet just based on the IP address and ignore protocol and ports.

Reply:
Yeah on that part I was not 100% sure